aten/src/ATen/native/mkldnn/Utils.h

```diff
@@ -21,4 +22,29 @@ inline bool mkldnn_bf16_device_check() {
       && cpuinfo_has_x86_avx512vl() && cpuinfo_has_x86_avx512dq();
 }
 
+inline bool mkldnn_bf16_gemm_usable_check(
+    const Tensor& mat1,
+    const Tensor& mat2,
+    const c10::optional<Tensor>& result_opt) {
+  c10::MaybeOwned<Tensor> result_maybe_owned = at::borrow_from_optional_tensor(result_opt);
+  const Tensor& result = *result_maybe_owned;
+
+  const int64_t mkldnn_gemm_min_size = 32 * 32 *32;
```

Comment:
the given benchmark shows even for 32^3, mkldnn bf16 is faster then ATen native impl, just it is slower than aten fp32 (which is mkl), why we need such a threshold here?

also check libxsmm bf16 perf to double confirm whether mkldnn counterpart need to be further improved.

Reply:
Changed it to 16^3